app/src/androidTest/java/HomePage.kt

```diff
@@ -0,0 +1,8 @@
+import com.bijoysingh.quicknote.R
+
+open class HomePage : EspressoBaseTest() {
```

Comment:
Myślę, że to mogę zmienić na object

Reply:
It shouldn't be object. You don't want to have single, shared test class for e.g. 2 or 5 Home screens. It shouldn't be open, because you are not extending `HomePage`